### PR TITLE
TB Management -  Tuberculosis Suspect diagnosis at Acute Illness

### PR DESCRIPTION
**Pull request by anvmn**
#1051
#1055

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2024-03-11 23-52

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -21162,6 +21183,12 @@ translationSet trans =
         TuberculosisReasonMedicationsNotDistributed reason ->
             translationSet <| TuberculosisReasonNotProvidedToday reason
 
+        TuberculosisSuspect ->
+            { english = "Tuberculosis Suspect"
+            , kinyarwanda = Nothing
+            , kirundi = Just "Nta Gupima/gusuzuma"
```
Comment (AronNovak) [inline client/src/elm/Translate.elm]: This is not the equivalent of the English string. This means: `No Testing/Assessment`

Comment (anvmn) [inline client/src/elm/Translate.elm]:
```suggestion
            , kirundi = Nothing
```

**Review by AronNovak — APPROVED** [review]

File: client/src/elm/Translate.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Translate.elm]: Thanks @AronNovak. Looks like a result of copy/paste and forgetting to replace values with Nothing.

**Pull request merged**